Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14.18.0-stretch-slim
+LABEL maintainer="Andrey Ivanov (https://github.com/343dev)"
+
+WORKDIR /app
+
+COPY . .
+
+ENV NODE_ENV="production"
```

Comment:
In our case, the variable `NODE_ENV="production"` allows `npm ci` not to install packages from the devDependencies section.

This will save 43 MB.